src/ckb-daemon/usb_linux.c

```diff
@@ -65,9 +65,9 @@ static char kbsyspath[DEV_MAX][FILENAME_MAX];
 /// If DEBUG_USB is set during compilation,
 /// the number of bytes sent and their representation are logged to the error channel.
 ///
-int os_usbsend(usbdevice* kb, const uchar* out_msg, int is_recv, const char* file, int line){
+int os_usbsend(usbdevice* kb, const uchar* out_msg, int is_recv, const char* file, int line) {
     int res;
-    if(kb->fwversion >= 0x120 && !is_recv){
+    if (kb->fwversion >= 0x120 && !is_recv) {
         struct usbdevfs_bulktransfer transfer;
```

Comment:
There's a neat way to initialize structs:
```c
struct usbdevfs_bulktransfer transfer = {0};
```
instead of `memset()`.

The C Standard guarantees this will initialize all members to their default values (e.g. integers to `0`, pointers to `NULL`). Also, using just empty braces `= {};` is also possible, although it's a GNU extension. But since it is already used in `device.c` (triple dots to set array boundaries), it should cause no problem, I suppose `clang` (even Apple's one) implements them.